### PR TITLE
Feat: add logging in handlers

### PR DESCRIPTION
**Pull request by daniel1sender**
In this PR I focused on implementing the logging in my HTTP layer using the logrus library. I created new logs and injected them in handlers of accounts and transfers.

### CODE REVIEW

File: pkg/domain/entities/transfer.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 var (
 	ErrAmountLessOrEqualZero = errors.New("amount is less or equal zero")
 	ErrSameAccountTransfer   = errors.New("transfer attempt to the same account")
+	ErrEmptyOriginID         = errors.New("invalid origin id")
+	ErrEmptyDestinationID    = errors.New("invalid origin id")
```
Comment (CMedrado) [inline pkg/domain/entities/transfer.go:14]: I think you wanted to put "invalid destination id" here

Comment (daniel1sender) [reply pkg/domain/entities/transfer.go:14]: you are right, thank you for your attention!

[the remark this answers is not in the document]
Comment (daniel1sender) [reply pkg/domain/entities/transfer.go:14]: you are right, thank you for your attention!

File: pkg/domain/transfers/make.go [hunk 2]
```diff
@@ -15,7 +16,7 @@ func (tu TransferUseCase) Make(originID, destinationID string, amount int) (enti
 	transfer, err := entities.NewTransfer(originID, destinationID, amount)
 
 	if err != nil {
-		return entities.Transfer{}, ErrCreatingNewTransfer
+		return entities.Transfer{}, fmt.Errorf("error when creating a transfer: %w", err)
```
[the remark this answers is not in the document]
Comment (daniel1sender) [reply pkg/domain/transfers/make.go:19]: nice catch! this is not making sense

File: pkg/gateways/http/accounts/create_test.go [hunk 3]
```diff
@@ -11,19 +11,22 @@ import (
...
 
 		account := entities.Account{Name: "Jonh Doe", CPF: "12345678910", Secret: "123", Balance: 0, CreatedAt: time.Now()}
 		useCase := accounts.UseCaseMock{Account: account}
-		h := NewHandler(&useCase)
-
+		h := NewHandler(&useCase, log)
 		createRequest := CreateRequest{account.Name, account.CPF, account.Secret, account.Balance}
 		request, _ := json.Marshal(createRequest)
 		newRequest, _ := http.NewRequest("POST", "/anyroute", bytes.NewReader(request))
+		newRequest.Header.Add("Request-Id", "request-id")
```
[the remark this answers is not in the document]
Comment (daniel1sender) [reply pkg/gateways/http/accounts/create_test.go:29]: I think I understand, this way is repetitive. I created a new constant to replace the second parameter in add method.

**Pull request closed without merging**